### ISSUE TITLE
all img.shields.io badges generate error messages

### ISSUE THREAD

**Issue by kalimu** [RMHogervorst/badgecreatr#25]
[![minimal R version](https://img.shields.io/badge/R>=-3.3.1-6666ff.svg)](https://cran.r-project.org/)

results in:

pandoc.exe: Could not fetch https://img.shields.io/badge/R%3E=-3.3.1-6666ff.svg
TlsExceptionHostPort (HandshakeFailed (Error_Protocol ("expecting server hello, got alert : [(AlertLevel_Fatal,HandshakeFailure)]",True,HandshakeFailure))) "img.shields.io" 443
Error: pandoc document conversion failed with error 67

**Comment by privefl** [comment]
You should give your `sessionInfo()` along with your reproducible example.

I suspect you use Windows. I had a similar problem, which seems to be a bug with Pandoc on Windows.

To solve my problem, I just added `<!-- rmarkdown v1 -->` at the top of my Rmd file. 

There may be a cleaner way to do this, but for now, I'm just waiting for this bug to be corrected in Pandoc. Lazy lazy :-)

**Comment by RMHogervorst** [comment]
@kalimu is this still a problem? I see the links are working on github and I have no errors on my machine (windows 64bit) . 

Thanks for the reporting! and thanks @privefl  for the speedy reaction

**Comment by RMHogervorst** [comment]
This still keeps happening in my projects as well 
Perhaps a try catch action?

**Comment by jsta** [comment]
This is a problem for me but `privefl `'s solution works.

**Comment by RMHogervorst** [comment]
Weird. What windows version are you using? And are all your packages up to date?

**Comment by jsta** [comment]
I only get the error if I have `badgecreatr` loaded via `library()`. If I restart RStudio (unloading `badgecreatr`) then I don't need `privefl`s solution.

----

> sessionInfo()
R version 3.4.0 (2017-04-21)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 16.04.2 LTS

Matrix products: default
BLAS: /usr/lib/libblas/libblas.so.3.6.0
LAPACK: /usr/lib/lapack/liblapack.so.3.6.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8    LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] badgecreatr_0.1.0    wql_0.4.9            devtools_1.13.1.9000

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.11        knitr_1.16          magrittr_1.5        pkgload_0.0.0.9000  munsell_0.4.3      
 [6] colorspace_1.3-2    lattice_0.20-35     rlang_0.1.1.9000    stringr_1.2.0       plyr_1.8.4         
[11] tools_3.4.0         pkgbuild_0.0.0.9000 grid_3.4.0          gtable_0.2.0        withr_1.0.2        
[16] htmltools_0.3.6     yaml_2.1.14         lazyeval_0.2.0      rprojroot_1.2       digest_0.6.12      
[21] tibble_1.3.3        reshape2_1.4.2      ggplot2_2.2.1.9000  codetools_0.2-15    evaluate_0.10      
[26] memoise_1.1.0       rmarkdown_1.5       stringi_1.1.5       compiler_3.4.0      scales_0.4.1       
[31] backports_1.1.0     zoo_1.8-0   

----
pandoc 1.16.0.2

**Comment by RMHogervorst** [comment]
I still think it is mostly a pandoc issue and not related to badgecreatr. Although it does impact this package. 

https://support.rstudio.com/hc/en-us/community/posts/202794236-Bump-pandoc-version-number 

I believe the latest version of rstudio has version 1.19 that should fix your bug. Please report back if it doesn't. I will try to find time this weekend

**Comment by jsta** [comment]
Ok, I upgraded my pandoc and RStudio (although I'm pretty sure `pandoc` does not ship with the linux `RStudio` builds). Still seeing error 67 with `badgecreatr` attached.

`pandoc --version`

> pandoc 1.19.2.1

RStudio version: 1.1.263

**Comment by RMHogervorst** [comment]
Thank you for testing! I'll let you know what I find this weekend

**Comment by jsta** [comment]
My best guess is a pandoc issue:
https://stackoverflow.com/questions/44136481/render-code-for-google-chart-in-rmarkdown-rmd

**Comment by RMHogervorst** [comment]
Don't be afraid to upen up the issue again if the new version of badge creatr (and pandoc) gives you this error again, Josheph @jsta .

**Comment by aravind-j** [comment]
I am facing the same problem with pandoc 1.19.2.1 in windows.
Not getting any badges for license, minimal R version, packageversion and last change date.

Only the badges for repo status and CRAN work.

**Comment by RMHogervorst** [comment]
Thanks for reporting!

**Comment by jmgirard** [comment]
Having this problem again:

```
---
title: "mwe"
output: html_document
---

[![lifecycle](https://img.shields.io/badge/lifecycle-maturing-blue.svg)](https://www.tidyverse.org/lifecycle/#maturing)
```

pandoc.exe: Could not fetch https://img.shields.io/badge/lifecycle-maturing-blue.svg
TlsExceptionHostPort (HandshakeFailed Error_EOF) "img.shields.io" 443
Error: pandoc document conversion failed with error 67
Execution halted

```
R version 3.5.1 (2018-07-02)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252 
[2] LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods  
[7] base     

other attached packages:
[1] usethis_1.3.0

loaded via a namespace (and not attached):
 [1] compiler_3.5.1  backports_1.1.2 magrittr_1.5    rprojroot_1.3-2
 [5] htmltools_0.3.6 tools_3.5.1     yaml_2.2.0      Rcpp_0.12.18   
 [9] stringi_1.2.4   rmarkdown_1.10  knitr_1.20      stringr_1.3.1  
[13] digest_0.6.15   evaluate_0.11  
```

**Comment by jlmelville** [comment]
Not sure if this is the most relevant place for this, but I found this issue while googling for the same pandoc error that others are reporting.

I am using Windows 10 (64-bit) and RStudio 1.1.463. I installed a new version of pandoc from https://github.com/jgm/pandoc/releases/latest (2.5 at the time of writing) and copied `C:\Program Files\Pandoc\pandoc.exe` to overwrite `C:\Program Files\RStudio\bin\pandoc\pandoc.exe`. 

"Upgrading" pandoc in this way worked for my needs (previewing markdown in RStudio).
